src/ui/server/helpers/task-list/task-helpers.ts

```diff
@@ -79,14 +81,16 @@ export const taskIsComplete = (taskFields: Array<string>, submittedData: Submitt
  * @param {Object} submittedData Submitted application data
  * @returns {Boolean}
  */
-export const areTaskDependenciesMet = (dependencies: Array<string>, submittedData: SubmittedDataInsuranceEligibility): boolean => {
+export const areTaskDependenciesMet = (dependencies: Array<string>, submittedData: ApplicationFlat): boolean => {
   const totalDependencies = (dependencies && dependencies.length) || 0;
 
   let validDependencies = [];
 
   if (dependencies) {
     validDependencies = dependencies.filter((fieldId: string) => {
-      if (submittedData && submittedData[fieldId]) {
+      // NOTE this assumes that any data in submitted fields, is a valid answer. E.g, "false" is a valid answer.
+
+      if (submittedData && (submittedData[fieldId] || submittedData[fieldId] === false)) {
```

Comment:
`submittedData[fieldId] || submittedData[fieldId] === false` <= worth making a helper function for this since we are using it more than once?

Reply:
Great shout

Reply:
Extracted this into it's own function, `hasSubmittedField`